docs/getting-started/openshift/index.md

```diff
@@ -0,0 +1,180 @@
+# ArgoCD Agent Hub and Spoke Cluster Setup Documentation Using Operator
+This document outlines the process of setting up ArgoCD Agent in a hub and spoke cluster architecture. This configuration allows a central ArgoCD instance (the hub) to manage applications deployed across multiple remote Kubernetes clusters (the spokes) through the argocd-agent.
+## Prerequisites
+Before proceeding with the setup, ensure you have the following:
+
+A running openshift/kubernetes cluster designated as the hub, with ArgoCD/openshift GitOps Operator installed.
+One or more Kubernetes clusters designated as spokes, ArgoCD/openshift GitOps Operator installed.
+oc configured to access both hub and spoke clusters.
+Operator must be installed in cluster scope mode.
```

Comment:
The Argo CD instance must be in cluster scoped mode right ? Operator is by default in cluster scoped (installs in All Namespaces)

Reply:
I was trying out autonomous mode without making it cluster scoped. Seemed to be working. I need to try it out again on the new staging build for confirming because previously I was facing issues with createNamespace not working and I didn't try it out after

Reply:
can it be reworded ? instead of saying `Operator` mention `Argo CD instance must be cluster scoped`.

Reply:
update